### DISCUSSION TITLE
撰写问题回答：训练好的深度学习模型是怎么部署的？

### DISCUSSION THREAD

**Discussion by OpenMMLab-Assistant-004** [open-mmlab/OpenMMLabCamp#700] [Other]
# 撰写问题回答：训练好的深度学习模型是怎么部署的？ 

要求：
1、图文并茂 不少于800字
2、要求内容与 OpenMMLab 算法库有关系，可以引导读者通过 OpenMMLab 框架来入门
3、先将内容撰写在共享文档内，将共享文档链接贴在 Discussion 评论区
4、Review 通过后，将由 OpenMMLab 官方账号发布在知乎

## Task Information

**Repository**：Other

**Difficulty**：Easy

**Points**：20

**Estimated Time Consumption**：14 days

## Task Status

**Claimant's Github ID**：ZhaoCake

**Claim Time**：2023年10月31日

**PR Link**：None

## Points to Note

To claim the task, please reply `Claim the mission` in the comment section. After claiming the task, please add our assistant bot called MeowMewo on WeChat at: `openmmlabwx`.

After completing the task, please submit a PR with the title starting with [MMSIG]. After submission, please reply with your PR link in the comments section of this Discussion. We will review your work as soon as possible.

在评论区回复 `Claim the mission` 领取任务，领取任务后请及时添加小助手喵喵微信: `openmmlabwx`

完成任务后提交相关链接（如是PR，标题请以[MMSIG]开头）回复到本Disucssion评论区，我们会第一时间审核您的贡献。

**Comment by ZhaoCake** [comment]
Claim the Mission

**Comment by ZhaoCake** [comment]
https://scurm-hotpot.feishu.cn/docx/TxFydIB69oDNe7xSBK0cARe4nSe?from=from_copylink

**Reply by ZhaoCake** [reply]
昨天分享链接的时候没有选择互联网上获得链接的人可阅读，现在可以了。  (*꒦ິ⌓꒦ີ)
